### PR TITLE
change ConfiguredModules to modules

### PR DESCRIPTION
**Pull request by michalgold**
change ConfiguredModules to modules

### CODE REVIEW

**Comment by michalgold** [comment]
Please note that I did not verify this yet.

File: Playbooks/playbook-C&C-Communication-Hunting.yml [hunk 1]
```diff
@@ -731,7 +731,6 @@ tasks:
       - "32"
     scriptarguments:
       brandname: ""
-      value: ${ConfiguredModules.brand(val=='Twilio')}
```
Comment (dorsha) [inline Playbooks/playbook-C&C-Communication-Hunting.yml]: why did you remove this?

**Comment by dorsha** [comment]
Don't forget to verify and add release notes.
